include/onnxruntime/core/framework/ml_value.h

```diff
@@ -83,6 +85,10 @@ struct OrtValue {
     fence_ = v.fence_;
   }
 
+  bool HasElement() const noexcept {
```

Comment:
>HasElement [](http://example.com/codeflow?start=7&length=10)

What does this method do? OrtValue does not know anything about elements

Reply:
It basically checks whether the OrtValue contains data. The naming of this method will be changed (Scott's comment already recommended this)

Reply:
Could IsAllocated() be used instead?

Reply:
No it cannot be. An OrtValue can be allocated but contain no data. This is the API to query for data existence.